sphinx_ext/make_docs_utilities.py

```diff
@@ -4,6 +4,7 @@
 import os
 import contextlib
 from typing import IO, Generator
+from deephaven_autodoc import AUTOFUNCTION_COMMENT_PREFIX
```

Comment:
Shouldn't this be `from .deephaven_autodoc` ?
```suggestion
from .deephaven_autodoc import AUTOFUNCTION_COMMENT_PREFIX
```

Reply:
this is how you import if there's no `__init__.py` in the directory (and there doesn't really need to be in this case)